src/EA.Weee.Web/Areas/Producer/ViewModels/EditContactDetailsViewModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace EA.Weee.Web.Areas.Producer.ViewModels
+{
+    using EA.Weee.Core.Organisations;
+    using EA.Weee.Core.Organisations.Base;
+    using System;
+    using System.Collections.Generic;
+
+    public class EditContactDetailsViewModel
+    {
+        public ContactDetailsViewModel ContactDetails { get; set; } = new ContactDetailsViewModel();
+        public Guid DirectRegistrantId { get; set; }
+
+        public Guid OrganisationId { get; set; }
+
+        public List<AdditionalContactModel> AdditionalContactModels { get; set; }
```

Comment:
@stevelee-civica dont think you need this, this is for the organisation model.

Reply:
removed